sourcegraph/metadata.csv

```diff
@@ -0,0 +1 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
```

Comment:
My apologies! I realized that you're using manifest v2 so you can remove this file.